home.md

```diff
@@ -0,0 +1,75 @@
+<img width="500" align="right" alt="illustration" src="https://user-images.githubusercontent.com/82679007/150655220-d023b26e-ceab-4b37-9d66-0ad6b20f0628.png">
```

Comment:
Could you put the image directly in the repository? You can create a folder `assets/images` and put all images in there

Reply:
Can you use the Markdown syntax to add images?  `![Alt text](path/to/image)`

Reply:
@Rahmot already created a folder for the images. I didn't want to create a duplicate file that's why I put it like this. I'll edit it once her PR is merged. Is that okay? or should I go-ahead to create an image folder?

Reply:
You can go ahead. It should not conflict with @rahmot

Reply:
Okay, I'll do that. Thank you :)